cosmo_tester/framework/cluster.py

```diff
@@ -121,17 +158,114 @@ def verify_services_are_running(self):
                     'service {0} is in {1} state'.format(
                             service['display_name'], instance['SubState'])
 
+    @retrying.retry(stop_max_attempt_number=10, wait_fixed=5000)
+    def assert_snapshot_created(self, snapshot_id, attributes):
```

Comment:
Yep. I think I assumed it might need different implementations per manager version, but that hasn't been the case. We can always move it back if that happens with some future version...